Library/ENV/4.3/xcrun

```diff
@@ -3,16 +3,18 @@
 # work at all (e.g. CLT-only in the Xcode 4.3 era). This script emulates
 # it and attempts to avoid these issues.
 
-# Some build tools set DEVELOPER_DIR, so discard it
-unset DEVELOPER_DIR
+if [ "$(uname -s)" == "Darwin" ]; then
```

Comment:
`-s` is the default. Please remove it. For short commands like this, I prefer backticks.